travel-journal/index.md

```diff
@@ -0,0 +1,910 @@
+---
+title: 'Personal Travel Journal with Vue.js and Directus'
+description: 'This tutorial covers how to create a personal travel journal with Directus and Vue.js with authentication features'
+author:
+  name: 'Mercy bassey'
+  avatar_file_name: 'mercy-bassey.jpg'
+---
+
+## Introduction
+
+Vue.js, a progressive JavaScript framework known for its simplicity and flexibility, has gained widespread popularity among developers for building interactive user interfaces. In this tutorial, you we’ll build a personal travel journal application using Vue.js in conjunction with Directus, an open-source headless CMS. In this guide we’ll go through the process of implementing user authentication with Directus and demonstrate how to interact with a Directus collection using the Vue.js framework.
+
+## Before You Start
+
+You'll need:
+
+- [Node.js](https://nodejs.org/en/) and a [text editor](https://kinsta.com/blog/best-text-editors/) installed on your machine.
+- A Directus project created either using [Directus Cloud](https://directus.cloud/) or [run locally on your machine.](https://docs.directus.io/getting-started/quickstart.html) This tutorial on the otherhand uses Directus Cloud.
+- Some knowledge of the Vue.js Composition API.
+
+## Creating A Vue.js Project
+
+Open up your command prompt or terminal app and run the following command:
+
+This will create a Vue.js project using [Vite](https://vitejs.dev/):
+
+```bash
+npm create vue@3
+```
+
+During project creation, when prompted with questions, choose the following configurations:
+
+```bash
+✔ Project name: … ./directus-travel-journal
+✔ Package name: … -directus-travel-journal
+✔ Add TypeScript? … No / Yes
+✔ Add JSX Support? … No / Yes
+✔ Add Vue Router for Single Page Application development? … No / Yes
+✔ Add Pinia for state management? … No / Yes
+✔ Add Vitest for Unit Testing? … No / Yes
+✔ Add an End-to-End Testing Solution? › No
+✔ Add ESLint for code quality? … No / Yes
+```
+
+Once completed, execute the following commands sequentially to finalize the project creation:
```

Comment:
```suggestion
Once completed, run the following command to install dependencies and start the development server:
```

It's always good to be more straightforward and remove words where they don't have impact. Execute -> Run, for example, is more clear.